openfisca_france/reforms/aah_deconjugalisee.py

```diff
@@ -0,0 +1,227 @@
+from openfisca_france.model.base import (
+    ADD, 
+    calculate_output_add, 
+    Famille,
+    Individu,
+    max_, 
+    min_, 
+    MONTH,
+    Reform, 
+    set_input_divide_by_period, 
+    Variable, 
+    where
+    )
+
+from numpy import datetime64
+
+
+class aah_deconjugalisee(Reform):
```

Comment:
Est-ce-qu'on ne voudrait pas renommer celle classe `aah_deconjugalisee_reform` afin de la différencier de la variable OF qu'on déclare en-dessous ?

Reply:
j'y avais pensé au départ, mais c'est raccord avec la façon d'écrire des Reform dans OF, cf. `allocations_familiales_imposables`